### ISSUE TITLE
Missing Calls

### ISSUE THREAD

**Issue by robertmeta** [robertmeta/swiftmac#27]
@parhamdoustdar can we search the codebase for any other things being sent to the backend that I might be missing, the set_lang for example I had no clue about, is there a regexp or anything we could run to try to find a complete list like you did for the tones?

Thanks!

**Comment by robertmeta** [comment]
I think I found most of them, some weird ones, set next langauge, set previous langauge, etc.

**Comment by parhamdoustdar** [comment]
My initial suggestion would be to open dtk-speak.el and search for `process-send-string`, but my assumption is that you've already got all of those. There's also a couple more in `emacspeak-sounds.el` which you probably know about from what I've gathered.
If all of those are already covered, once #26 is finished, how about @tvraman having a look for completeness? That way we can hit two birds with one stone; we can make sure the docs are good and that we have all the commands covered.

**Comment by robertmeta** [comment]
I actually already patched the emacspeak docs, so this is good to go.
